server/routes/hdc.js

```diff
@@ -72,22 +77,24 @@ module.exports = function({logger, licenceService, conditionsService, authentica
         res.render(`licenceConditions/conditionsSummaryForm`, {nomisId, licence, nextPath});
     }));
 
-    router.get('/:formName/:nomisId', asyncMiddleware(async (req, res) => {
-        const {nomisId, formName} = req.params;
+    // standard routes
+
+    router.get('/:sectionName/:formName/:nomisId', asyncMiddleware(async (req, res) => {
+        const {sectionName, formName, nomisId} = req.params;
         logger.debug(`GET licenceConditions/${formName}/${nomisId}`);
 
         const rawLicence = await licenceService.getLicence(nomisId);
         const {licenceSection, nextPath, licenceMap} = formConfig[formName];
-        const dataPath = licenceMap || ['licence', 'licenceConditions', licenceSection];
+        const dataPath = licenceMap || ['licence', sectionName, licenceSection];
         const data = getIn(rawLicence, dataPath) || {};
 
-        res.render(`licenceConditions/${formName}Form`, {nomisId, data, nextPath});
+        res.render(`${sectionName}/${formName}Form`, {nomisId, data, nextPath});
```

Comment:
we still have to think about maybe moving `${formName}Form` to the config?

Reply:
Agreed